systest/cluster/cluster.go

```diff
@@ -50,6 +52,14 @@ func defaultBootnodes(size int) int {
 	return bsize
 }
 
+func defaultRemote(size int) int {
+	remote := (size / 4)
+	if remote == 0 {
+		return 1
+	}
+	return remote
+}
```

Comment:
It's cryptic what this is for and why divide by 4 :thinking:. Perhaps this shouldn't be a function at all? Inlining it would give more context.

Also, consider 
```suggestion
func defaultRemote(size int) int {
	return max(1, size/4)
}
```

Reply:
I just did the same thing as with `defaultBootnodes`. I can change it to `max`, its a good simplification 🙂

I can add a comment explaining that 25% of all nodes by default are deployed with a remote Post Service. Arguably this could be a setting additional to `testcontext.ClusterSize` which defaults to 1/4 of it, but then we would need code to check that "`testcontext.RemoteSize`" is compatible to `testcontext.ClusterSize`. But maybe we should wait with changing this until we want to run tests with multi-smesher nodes? Wdyt?

Reply:
Maybe the default should be zero and spawn remote services only in a dedicated test?

Reply:
I changed the default size to zero and changed `TestSmeshing` to run 25% of all nodes in remote smeshing